cerbero/commands/package.py

```diff
@@ -44,7 +44,7 @@ def __init__(self):
             ArgparseArgument('-f', '--force', action='store_true',
                 default=False, help=_('Delete any existing package file')),
             ArgparseArgument('-d', '--no-devel', action='store_false',
-                default=True, help=_('Do not create the development version '
+                default=False, help=_('Do not create the development version '
```

Comment:
Actually we both assumed it wrong, check https://docs.python.org/2/library/argparse.html
The store_true is like store but also sets a default value, to True in this case. So or we either use everywhere just store where a default value is set, or store_true/store_false and remove the default option.